requirements.txt

```diff
@@ -15,3 +13,10 @@ coverage
 awscli
 flake8
```

Comment:
Are these requirements needed? I guess they were added after default generation of this file in the very begining

Reply:
Could you remove all not needed libs?

Reply:
Updated.